src/static/containers/ShortCard.js

```diff
@@ -5,19 +5,37 @@ import { bindActionCreators } from 'redux';
 import PropTypes from 'prop-types';
 import { Icon, Rate, Col, Row, Card, Button } from 'antd';
 
-import * as actionCreators from '../actions/cards';
+import { deleteCard } from '../actions/cards';
+import editUserCard from '../actions/userCards';
 
 class ShortCard extends Component {
-    deleteCard = () => {
+    delCard = () => {
         const { actions: { deleteCard }, token, card: { slug } } = this.props;
         deleteCard(token, slug);
     };
 
     goToEditCard = () => {
         const { card: { slug }, dispatch } = this.props;
         dispatch(push(`/zw-admin/card-edit/${slug}`));
+    };
+
+    changeCardStatus = (cardStatus) => {
+        const {
+            refreshUserCards,
+            actions: { editUserCard },
+            token,
+            card: { id },
+        } = this.props;
+        editUserCard(token, id, { status: cardStatus }, refreshUserCards);
```

Comment:
Maybe use a different name because this is already the name of the action from the import